stock_cycle_count/models/stock_warehouse.py

```diff
@@ -40,8 +39,7 @@ def get_horizon_date(self):
     @api.model
     def _get_cycle_count_locations_search_domain(
             self, parent):
-        domain = [('parent_left', '>=', parent.parent_left),
-                  ('parent_right', '<=', parent.parent_right),
+        domain = [('parent_path', '=like', parent.parent_path + '%'),
```

Comment:
I'm wondering why child of/parent of would not works here ?

Reply:
@robertrottermann I use it as a reference https://github.com/odoo/odoo/blob/12.0/addons/stock/models/product.py#L206 and https://github.com/odoo/odoo/blob/12.0/addons/stock/models/product.py#L211

Reply:
Yeah, that's weird because it's in the same commit that implements parent_path to optimize queries...

Reply:
Additionally, parent_left and parent_right fields are removed from stock.location object.

Reply:
Yes, that was the aim of parent_path: to optimize child of/parent of search operators, so, normally, you should use them as before.

Reply:
so what do you recommend on these ?